.env.babylon-integration.example

```diff
@@ -11,16 +11,11 @@
 
 ### Bitcoin Configuration
 BITCOIN_NETWORK=signet
-BITCOIN_RPC_HOST=11.22.33.44:38332 # gRPC
-BITCOIN_RPC_USER=snapchain
+BITCOIN_RPC_IP=11.22.33.44
+BITCOIN_RPC_PORT=38332
 BITCOIN_RPC_PASS=rpcpass
-# TODO: there is no need to make these ports configurable. should just have one BITCOIN_NODE_IP var
-# and use it here and for BITCOIN_RPC_HOST 
-ZMQ_RAWBLOCK_URL=tcp://11.22.33.44:29001
-ZMQ_RAWTR_URL=tcp://11.22.33.44:29002
 
 # Bitcoin Wallet Configuration
-BTC_WALLET_NAME=btcwallet
```

Comment:
removing these doesn't simply our codebase, it introduces lots of copy-paste. i think we should revert all of the changes

Reply:
the TODO i added here is only to see if we can simplify

```
ZMQ_RAWBLOCK_URL=tcp://11.22.33.44:29001
ZMQ_RAWTR_URL=tcp://11.22.33.44:29002
```

not to make the codebase more coupled

Reply:
we decided to make these ports non-configurable and use the default ports

Reply:
we shouldn't introduce sth like this.

<img width="1211" alt="image" src="https://github.com/user-attachments/assets/1a825229-dabc-42d8-a409-efe1890222a7">

it's better to have one place for source of truth

Reply:
likewise, for

```
ZMQ_RAWBLOCK_URL=tcp://11.22.33.44:29001
ZMQ_RAWTR_URL=tcp://11.22.33.44:29002
```

we should use the default port but we shouldn't introduce multiple source of truth either

<img width="1331" alt="image" src="https://github.com/user-attachments/assets/eacb3c9e-e2c5-4cd5-aa67-163d6e1b9fcb">

if we cannot find a way to do it, it's better to keep it as it is

Reply:
got it

Reply:
note that the devops PR is still good. e.g. we keep BTC_WALLET_NAME in the env file here but we can remove it in the ansible ini

b/c the env file in this repo doesn't need to be the same as the ini file in the other repo